CorsixTH/Lua/graphics.lua

```diff
@@ -432,52 +436,34 @@ function Graphics:loadLanguageFont(name, sprite_table, ...)
       cache[sprite_table] = font
     end
   end
-  self.load_info[font] = {self.loadLanguageFont, self, name, sprite_table, ...}
+  self.load_info[font] = {self.loadLanguageFont, self, name, sprite_table, x_sep, y_sep, ttf_col, force_bitmap}
   return font
 end
 
-function Graphics:loadFont(sprite_table, x_sep, y_sep, ...)
-  -- Allow (multiple) arguments for loading a sprite table in place of the
-  -- sprite_table argument.
-  local load_font = x_sep
-  if type(sprite_table) == "string" then
-    local arg = {sprite_table, x_sep, y_sep, ...}
-    local n_pass_on_args = #arg
-    for i = 2, #arg do
-      if type(arg[i]) == "number" then -- x_sep
-        n_pass_on_args = i - 1
-        break
-      end
-    end
-    sprite_table = self:loadSpriteTable(unpack(arg, 1, n_pass_on_args))
-    if n_pass_on_args < #arg then
-      x_sep, y_sep = unpack(arg, n_pass_on_args + 1, #arg)
-    else
-      x_sep, y_sep = nil, nil
-    end
-  end
+function Graphics:loadFontAndSpriteSheet(dir, name, complex, palette, x_sep, y_sep, ttf_col)
+  local sprite_table = self:loadSpriteTable(dir, name, complex, palette)
+  local force_bitmap = (name == "Font05V" and self:arabicNumerals())
 
-  local use_bitmap_font = true
-  if (not sprite_table:isVisible(46)) or (load_font == "Font05V" and self:arabicNumerals()) then -- luacheck: ignore 542
-    -- The font doesn't contain an uppercase M, so (in all likelihood) is used
-    -- for drawing special symbols rather than text, so the original bitmap
-    -- font should be used.
-  elseif self.language_font then
-    use_bitmap_font = false
-  end
+  return self:loadFont(sprite_table, x_sep, y_sep, ttf_col, force_bitmap)
+end
+
+function Graphics:loadFont(sprite_table, x_sep, y_sep, ttf_col, force_bitmap)
```

Comment:
I am assuming this is now a private function?

Reply:
It is not.